### PR TITLE
14691: Enrollment Data Clean Up

### PR DESCRIPTION
**Pull request by rabbittracey**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [14691](https://devops.dchbx.org/redmine/issues/14691) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | UP-Disqueued |
| Development Story Points | 2 |
| Peer Review Story Points | n/a |
| Ticket Author | @DFos85 |

### CODE REVIEW

File: lib/tasks/enrollment_data_clean_up.rake [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                    ]
+    CSV.open(file_name, "w") do |csv|
+      csv << field_names
+      families=Family.where(:"households.hbx_enrollments.aasm_state".in => HbxEnrollment::CANCELED_STATUSES)
+      families.each do |family|
+        enrollments=family.active_household.hbx_enrollments
+        family.active_household.hbx_enrollments.where(aasm_state:"coverage_canceled").each do |canceled_enrollment|
+          enrollments.each do |enrollment|
+            if canceled_enrollment.kind == enrollment.kind
+              cancel_member=canceled_enrollment.hbx_enrollment_members.where(is_subscriber:true).first
```
Comment (blackeaglejs) [inline lib/tasks/enrollment_data_clean_up.rake]: you can use "canceled_enrollment.subscriber" -> it'll be much cleaner.

File: lib/tasks/enrollment_data_clean_up.rake [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+              reference_member=enrollment.hbx_enrollment_members.where(is_subscriber:true).first
```
Comment (blackeaglejs) [inline lib/tasks/enrollment_data_clean_up.rake]: you can use enrollment.subscriber instead -> it'll be much cleaner.

File: lib/tasks/enrollment_data_clean_up.rake [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+              if cancel_member.id == reference_member.id
```
Comment (blackeaglejs) [inline lib/tasks/enrollment_data_clean_up.rake]: if one or the other of these don't exist, there will be a lot of failures.

File: lib/tasks/enrollment_data_clean_up.rake (hunk 1 above)
Comment (rabbittracey) [inline lib/tasks/enrollment_data_clean_up.rake]: agree, already modified

File: lib/tasks/enrollment_data_clean_up.rake (hunk 2 above)
Comment (rabbittracey) [inline lib/tasks/enrollment_data_clean_up.rake]: agree, modified

**Pull request merged**